android/app/src/main/AndroidManifest.xml

```diff
@@ -31,22 +31,26 @@
             android:name="flutterEmbedding"
             android:value="2" />
     </application>
+    <!-- Required to query activities that can process text, see:
+         https://developer.android.com/training/package-visibility and
+         https://developer.android.com/reference/android/content/Intent#ACTION_PROCESS_TEXT.
 
+         In particular, this is used by the Flutter engine in io.flutter.plugin.text.ProcessTextPlugin. -->
     <queries>
-        <!-- Required to query activities that can process text, see:
-            https://developer.android.com/training/package-visibility and
-            https://developer.android.com/reference/android/content/Intent#ACTION_PROCESS_TEXT.
-
-            In particular, this is used by the Flutter engine in io.flutter.plugin.text.ProcessTextPlugin. -->
-        <!-- <intent>
+        <intent>
             <action android:name="android.intent.action.PROCESS_TEXT"/>
             <data android:mimeType="text/plain"/>
-        </intent> -->
-
+        </intent>
+        <intent>
+            <action android:name="android.intent.action.PROCESS_TEXT"/>
+            <data android:mimeType="text/plain"/>
+        </intent>
         <!-- Launching URLs -->
         <intent>
             <action android:name="android.intent.action.VIEW" />
             <data android:scheme="https" />
         </intent>
     </queries>
-</manifest>
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
> Is this needed?
> From privacy perspective, I'd prefer if this app doesn't require Internet.

> Yes, it is needed. Please update it because it is required to fetch the 2FA data.

Note that TOTP and HOTP do not require Internet. 
The code is generated from a secret (scanned from QR code or manually inputted), and additionally for TOTP, the clock time.